create_infotexts.py

```diff
@@ -374,15 +375,44 @@ def process_depicted_place(self, place_string, places_mapping):
                 place_as_wikitext = place_string
 
             # Don't forget to add the commons categories, even though only wikidata is used in depicted people field
-                if places_mapping[place_string].get('commonscat'):
-                    self.content_cats.append(places_mapping[place_string]["commonscat"])
+            if places_mapping[place_string].get('commonscat') and \
+            places_mapping[place_string].get('commonscat') != "-" and \
+            places_mapping[place_string].get('commonscat') not in self.content_cats:
+
+                self.content_cats.append(places_mapping[place_string]["commonscat"])
 
         else:
             self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
             place_as_wikitext = place_string
 
 
+        for place in places_mapping:
```

Comment:
So a couple of questions here.
* Do we always want to run this second logic or only when we have not already found a mapped place?
** My gut feeling is the latter since the place field should be more reliable than the description.
** If the former. How do we prevent this from adding the same formatting a second time?

I'll stick some code related comments below but this one might affect whether the others are needed.